src/components/Articles.jsx

```diff
@@ -32,9 +37,9 @@ const Articles = () => {
             <h2>
                 Articles
             </h2>
-            <select name="topic_name" id="topic_name" defaultValue="select_topic">
+            <select name="topic_name" id="topic_name" defaultValue="show-all">
                 <option value="select_topic" disabled>Filter by topic...</option>
-                <option value="">Show All</option>
+                <option value="show-all">Show All</option>
                 <option value="cooking">Cooking</option>
                 <option value="coding">Coding</option>
                 <option value="football">Football</option>
```

Comment:
Ah, I tried to use this in the browser but it didn't do anything - but I can see you've not put any functionality in it yet to actually change the page. WIP I guess!